test/plugins/file-description-task-description-plugin-test.js

```diff
@@ -0,0 +1,48 @@
+const buster = require('buster')
+const plugin = require('../../lib/plugins/file-description-task-description-plugin.js')
```

Comment:
Das string-similarity package muss ich aber nicht nochmal testen, wenn es schon in similar-title-plugin-test getestet wird